packages/forma-36-react-components/src/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FunctionComponent, KeyboardEvent } from 'react';
+import Paragraph from '../Typography/Paragraph';
+import classNames from 'classnames';
+import styles from './style.css';
+
+export interface SwitchProps {
+  onToggle?: (checked: boolean) => void;
+  checked?: boolean;
+  labelText: string;
+  disabled?: boolean;
+  extraClassNames?: string;
+}
+
+export const Switch: FunctionComponent<SwitchProps> = (props: SwitchProps) => {
+  return (
+    <main
+      className={classNames(styles['Switch__wrapper'], props.extraClassNames)}
+    >
+      <aside
```

Comment:
Why are you using those semantic elements? Why does it make sense that the switch is an aside?

Reply:
To make it easier to understand the general structure; one container and an inner box on the side next to the paragraph